### DISCUSSION TITLE
v1.5.2

### DISCUSSION THREAD

**Discussion by Spacesai1or** [ethereum/web3.js#4243] [Announcements]
### Fixed

- Remove transaction `type` defaulting for `eth.sendTransaction`, `eth.sendRawTransaction` (#4241)
- `type: 0x0` was being added to legacy transaction when using `eth.signTransaction` (#4241)

<hr /><em>This discussion was created from the release <a href='https://github.com/ChainSafe/web3.js/releases/tag/v1.5.2'>v1.5.2</a>.</em>

**Comment by golashr** [comment]
Hi @spacesailor24 
We use web3-core `signTransaction` typescript for submit transaction. We realised that `TransactionConfig` interface used for this function, still doesn't have `maxPriorityFeePerGas` and `maxFeePerGas` fields yet. 
Questions
a. When can we expect this change to be available? 
b. When will the changes under https://github.com/ChainSafe/web3.js/issues/4283 be released?

**Reply by Spacesai1or** [reply]
Thank you for reaching out!

Not updating the typescript interface was an oversight, so we'll most likely bundle that in with the next release

Additionally I'm hoping to get an RC out this week once we get #4277 merged

**Reply by philknows** [reply]
We're aiming to get that out by the end of this week. Will keep you updated, thanks for reaching out!
